src/main/java/org/pac4j/vertx/core/engine/VertxCallbackLogic.java

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
I finally decided to remove all header license in source files and have two `NOTICE`  and `LICENSE` files instead at the root of the project: it makes only one place to upgrade to say it's 2016.

See: https://github.com/pac4j/j2e-pac4j/blob/master/NOTICE and https://github.com/pac4j/j2e-pac4j/blob/master/LICENSE